pkg/audit/cursor.go

```diff
@@ -93,6 +88,14 @@ func (cursor *Cursor) NextStripe(ctx context.Context) (stripe *Stripe, err error
 		return nil, err
 	}
 
+	maxSize := int64(es.StripeSize())
+
+	peerIdentity := &identity.PeerIdentity{ID: cursor.identity.ID, Leaf: cursor.identity.Leaf}
+	pba, err := cursor.allocation.PayerBandwidthAllocation(ctx, peerIdentity, pb.BandwidthAction_GET_AUDIT, maxSize)
```

Comment:
This will break audits. The PBA is limited to 1024 bytes but the RBA will have a size of 256000. Storage nodes will have to reject the invalid bwagreement.